themes/reactos/layouts/partials/nav.html

```diff
@@ -1,57 +1,4 @@
-<div class="navbar-affixed-top" data-spy="affix" data-offset-top="200">
-	<div class="navbar navbar-default yamm" role="navigation" id="navbar">
-		<div class="col-md-offset-1 col-md-10">
-			<div class="navbar-header">
-				<a class="navbar-brand home" href="{{ .Site.BaseURL }}">
-					<img src="{{ .Site.BaseURL }}{{ .Site.Params.logo }}" alt="{{ .Title }} logo"
-						class="hidden-xs hidden-sm">
-					<img src="{{ .Site.BaseURL }}{{ .Site.Params.logo_small }}" alt="{{ .Title }} logo"
-						class="visible-xs visible-sm">
-					<span class="sr-only">{{ .Title }} - {{ i18n "navHome" }}</span>
-				</a>
-				<div class="navbar-buttons">
-					<button type="button" class="navbar-toggle btn-template-main" data-toggle="collapse"
-						data-target="#navigation">
-						<span class="sr-only">{{ i18n "navToggle" }}</span>
-						<i class="fa fa-align-justify"></i>
-					</button>
-				</div>
-			</div>
-
-			<div class="navbar-collapse collapse" id="navigation">
-				<ul class="nav navbar-nav navbar-right">
-					{{ $current := . }}
-					{{ range .Site.Menus.main }}
-					{{ $topLevel := replace .URL "/" "" }}
-
-					<li
-						class="dropdown{{ if not $current.IsHome }}{{ if or (eq $current.RelPermalink .URL) (eq $current.Type $topLevel) }} active{{ end }}{{ end }}">
-						{{ if .HasChildren }}
-
-							<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{ .Name }}<span class="caret"></span></a>
-							<ul class="dropdown-menu">
-								{{ range .Children }}
-								<li><a href="{{ .URL }}">{{ .Pre }}{{ .Name }}{{ .Post }}</a></li>
-								{{ end }}
-							</ul>
-						{{ else }}
-							<a href="{{ .URL }}">{{ .Pre }}{{ .Name }}{{ .Post }}</a>
-						{{ end  }}
-					</li>
-					{{ end }}
-				</ul>
-			</div>
-
-			<div class="collapse clearfix" id="search">
-				<form class="navbar-form" role="search">
-					<div class="input-group">
-						<input type="text" class="form-control" placeholder="Search">
-						<span class="input-group-btn">
-							<button type="submit" class="btn btn-template-main"><i class="fa fa-search"></i></button>
-						</span>
-					</div>
-				</form>
-			</div>
-		</div>
-	</div>
-</div>
+<!-- Padding only -->
+{{ partial "navbar.html" . }}
+<!-- Actual navigation bar -->
+{{ partial "navbar.html" . }}
```

Comment:
Include the same file twice?

Reply:
Exactly. It's being distinguished by CSS `.navbar-affixed-top:nth-child(2)`.